<launch>
    <arg name="serial_no_camera1" default="134222070753" />
    <arg name="serial_no_camera2" default="132422070313" />
    <arg name="camera1" default="base_camera" />
    <arg name="camera2" default="side_camera" />
    <arg name="tf_prefix_camera1" default="$(arg camera1)" />
    <arg name="tf_prefix_camera2" default="$(arg camera2)" />
    <arg name="initial_reset" default="true" />
    <arg name="reconnect_timeout" default="6.0" />
    <arg name="publish_tf" default="false" />
    <arg name="publish_odom_tf" default="false" />
    <arg name="color_width" default="640" />
    <arg name="color_height" default="480" />
    <arg name="enable_color" default="true" />
    <arg name="color_fps" default="60" />


    <group ns="$(arg camera1)">
        <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
            <arg name="serial_no" value="$(arg serial_no_camera1)" />
            <arg name="tf_prefix" value="$(arg tf_prefix_camera1)" />
            <arg name="initial_reset" value="$(arg initial_reset)" />
            <arg name="reconnect_timeout" value="$(arg reconnect_timeout)" />
            <arg name="color_width" value="$(arg color_width)" />
            <arg name="color_height" value="$(arg color_height)" />
            <arg name="enable_color" value="$(arg enable_color)" />
            <arg name="color_fps" value="$(arg color_fps)" />
        </include>
    </group>

    <group ns="$(arg camera2)">
        <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
            <arg name="serial_no" value="$(arg serial_no_camera2)" />
            <arg name="tf_prefix" value="$(arg tf_prefix_camera2)" />
            <arg name="initial_reset" value="$(arg initial_reset)" />
            <arg name="reconnect_timeout" value="$(arg reconnect_timeout)" />
            <arg name="color_width" value="$(arg color_width)" />
            <arg name="color_height" value="$(arg color_height)" />
            <arg name="enable_color" value="$(arg enable_color)" />
            <arg name="color_fps" value="$(arg color_fps)" />
        </include>
    </group>

</launch>